planning/behavior_path_dynamic_avoidance_module/src/scene.cpp

```diff
@@ -733,7 +951,7 @@ LatFeasiblePaths DynamicAvoidanceModule::generateLateralFeasiblePaths(
   return ego_lat_feasible_paths;
 }
 
-void DynamicAvoidanceModule::updateRefPathBeforeLaneChange(
+[[maybe_unused]] void DynamicAvoidanceModule::updateRefPathBeforeLaneChange(
```

Comment:
Sorry, I don't remember correctly.
This part is related to fixing the conflict between LC and dynamic avoidance.
I assigned the ticket of the bug of the conflict, so when you work on it, please think about if this function is necessary or not.

Reply:
OK I got it